### PR TITLE
Allow renew to handle --webroot-path on the commandline

### PR DESCRIPTION
**Pull request by pde**
- and generally clarify how renew handles webroot options

Closes: #2424

### CODE REVIEW

File: letsencrypt/cli.py [hunk 1]
```diff
@@ -901,24 +922,15 @@ def _reconstitute(config, full_path):
...
-    # configuration restoring logic is not able to correctly parse it
-    # from the serialized form.
-    if "webroot_map" in renewalparams and not _set_by_cli("webroot_map"):
-        setattr(config.namespace, "webroot_map", renewalparams["webroot_map"])
-
     try:
-        domains = [le_util.enforce_domain_sanity(x) for x in
-                   renewal_candidate.names()]
+        for d in renewal_candidate.names():
+            _process_domain(config, d)
```
Comment (schoen) [inline letsencrypt/cli.py]: Are you sure that this correctly updates config in substantially the same way the old code did?

**Comment by schoen** [comment]
There was a missing colon in a function definition so I've pushed a fix to that to get the Travis tests started again.

**Comment by bmw** [comment]
Closing to make travis tests start

**Comment by schoen** [comment]
Aha!

**Pull request merged**